test/rejoice.js

```diff
@@ -221,14 +221,14 @@ describe('start()', function () {
                 expect(err).to.not.exist();
                 expect(server).to.exist();
 
-                server.start = function (callback) {
+                server.start = function (cb) {
 
                     Glue.compose = compose;
                     console.error = consoleError;
 
                     Fs.unlinkSync(configPath);
 
-                    callback();
+                    cb();
```

Comment:
I get why the rename, but do you even need the `callback` in here? Can you just call `done` and forgo the inner callback entirely?